README.md

```diff
@@ -200,6 +200,8 @@ You can easily serve your ML model from a remote server using `ML streaming` fea
 
 ⚠️ In order to use `ML streaming` feature, make sure you've installed the `streaming` mode of PyMilo
 
+⚠️ **Version compatibility note:** For ML streaming over `WebSocket`, please use PyMilo versions `<=1.3`. In version `1.4`, `ML streaming` over `WebSocket` is under construction and not fully functional.
```

Comment:
I suggest saying that this section is under construction and is not stable. It's a bit weird that it refers to a previous version.